core/templates/components/forms/custom-forms-directives/audio-file-uploader.component.ts

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { IdGenerationService } from 'services/id-generation.service';
+
+/**
+ * @fileoverview Directive that enables the user to upload audio files.
+ */
+
+@Component({
+  selector: 'audio-file-uploader',
+  templateUrl: './audio-file-uploader.component.html',
+  styleUrls: []
+})
+export class AudioFileUploaderComponent implements OnInit {
+  @Input() droppedFile?: FileList;
+  @Output() onFileChanged: EventEmitter<File> = new EventEmitter<File>();
```

Comment:
Don't prefix `on` for `@Output` variables.

Reply:
Done.